trieste/models/gpflow/models.py

```diff
@@ -469,9 +479,70 @@ def num_data(self, value: TensorType) -> None:
         self._num_data.assign(value)
 
 
-class SVGPWrapper(SVGP, NumDataPropertyMixin):
-    """A wrapper around GPFlow's SVGP class that stores num_data in a tf.Variable and exposes
-    it as a property."""
+class Parameter(gpflow.Parameter):
```

Comment:
can we move this class to util.py in this folder? this file is getting rather big...

Reply:
Late to the party, but wanted second this. Perhaps just separate parameter.py in the same folder would be appropriate for this class?